src/sentry/api/endpoints/organization_artifactbundle_assemble.py

```diff
@@ -56,19 +57,31 @@ def post(self, request: Request, organization) -> Response:
         except Exception:
             return Response({"error": "Invalid json body"}, status=400)
 
-        projects = set(data.get("projects", []))
-        if len(projects) == 0:
+        input_projects = set(data.get("projects", []))
```

Comment:
nit: I think we can skip converting this to a set as I don't see it being used anywhere